test/JDBC/upgrade/14_latest/schedule

```diff
@@ -40,3 +40,5 @@ BABEL-3234
 column_domain_usage
 constraint_column_usage
 sys-sysobjects
+Test-sp_helpsrvrolemember
+Test-sp_helpsrvrolemember-14_4
```

Comment:
this needs update as well, the test should have failed as nothing should match this

Reply:
Sorry I missed it. Updated the schedule file.